documentation/Windows-Installation-Guide.md

```diff
@@ -0,0 +1,399 @@
+![postgis rasters 2](https://user-images.githubusercontent.com/19670372/33494826-79fad25a-d6bc-11e7-8b63-1a35044e23d4.png)
+- Click **Close** when the installation is finished
+- Click **Finish** on **Stack Builder Installation Completed** screen
+
+###### Note: We still have to set the Environment Variable path, as psql command will still not work on Command Prompt
+
+##### Set Environment Variable
+
+- Navigate to the **bin** directory of postgresql installation, which in my case is 
+```C:\Program Files\PostgreSQL\10\bin```. Click on the address bar of the file explorer window and copy the path.
+- Navigate to the **Environment variable** window as described in the Java installation section, and when you are on 
+**Edit environment variable** screen, click on **New** and paste the address of the **Postgresql bin directory** that 
+you copied a little while ago and click **OK**  
+![Postgres set env](https://user-images.githubusercontent.com/19670372/33494822-799f2e1e-d6bc-11e7-9ddc-784b053dfb31.png)
+- Now open a new window of **Command Prompt** and type ```psql -U postgres``` as by default **Postgresql** creates a 
+**postgres** account, upon entering the above command it will prompt you to enter password for **postgres** user which 
+you have set while installing the Postgresql. Type the same password in and you should see something like below. 
+Screenshot below shows a Postgresql shell. Type ```\q``` to quit.  
+![Postgres installed](https://user-images.githubusercontent.com/19670372/33494910-81b6fa3c-d6bc-11e7-9302-e57a9d19a9bc.png)
+
+###### Congratulations now you have installed Postgres and Postgis.
+
+## Installing Git
+
+### Check if Git is already installed
+
+- Lets first check if Git is installed on your machine or not. Open a new **Command prompt** 
+window and type ```git --version```, if you see something like below then we could say that git is not 
+installed on your computer. It could be that Git is installed but **Environment Variable** is not set. 
+You could check that by going to the ```C:\Program Files``` if you would see a folder named **Git** then you just need 
+to set **Environment Variable** path. If that is the case please feel free to the section 
+where we are setting **Environment Variable**  
+![Git not installed](https://user-images.githubusercontent.com/19670372/33494892-8011399a-d6bc-11e7-82a9-7efc9b2bf130.png)
+
+### Download Git
+
+- Go [here](https://github.com/git-for-windows/git/releases/tag/v2.15.0.windows.1) and scroll down to **Downloads** 
+section and click the latest version of the application which has **.exe** extension, which in my case is 
+**Git-2.15.0-32-bit.exe**  
+![Git Download](https://user-images.githubusercontent.com/19670372/33494905-814dfa50-d6bc-11e7-9961-c469df209243.png)
+
+### Install Git
+
+- Double click on the file once it is downloaded and click **Run**.
```

Comment:
The following `Next` steps could be avoided by saying `Follow the installation steps` because they have nothing special in this case.

Reply:
Just wanted to be as specific as possible, if you think its a good idea to remove those steps, i would do that

Reply:
Cool!